Lesson16/Les16/gokce/script.js

```diff
@@ -0,0 +1,195 @@
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function comparethem(firstnumber,secondnumber){
+  if(firstnumber>secondnumber){
+    console.log('The first number is bigger then the second number');
+  }
+  else if(firstnumber<secondnumber){
+    console.log('The first number is less than the second number')
+  }
+  else{
+    console.log('Numbers are equal')
+  }
+}
+comparethem(39,90);
+comparethem(3,1);
+
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function numbertype(number){
+    if(number>0){
+        console.log('The number is positive');
+    }
+    else if(number<0){
+        console.log('The number is negative');
+    }
+    else{
+        console.log('The number is zero');
+    }
+}
+numbertype(0);
+numbertype(80);
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function yeartype(year){
+    if(year%4===0 || year%400===0){
```

Comment:
It's important to check that the number is both divisible by 4 and **not divisible by 100**
```suggestion
    if(year%4===0 && year % 100 !== 0 || year%400===0){
```